component/src/main/wso2/sap/Module.md

```diff
@@ -54,13 +38,13 @@ You can uninstall the module by executing either of the uninstall.{sh/bat} scrip
 ```ballerina
 
 sap:ProducerConfig producerConfigs = {
-    destinationName:"<The SAP gateway name>",
-    sapClient:"<SAP client, for example, 001>",
-    userName:"<The user logon>",
-    password:"<The logon password>",
-    asHost:"<The R/3 application server>",
-    sysnr:"<SAP system number, for example, 01>",
-    language:"<The logon language>"
+    destinationName: "<The SAP gateway name>",
+    ^"client: "<SAP client, for example, 001>",
+    username: "<The user logon>",
+    password: "<The logon password>",
+    ashost: "<The R/3 application server>",
```

Comment:
Changed the camel notation, please check and update in README.md too.

Reply:
As per the Rajkumar, customer wanted this notation for sap parameters.